pkg/v1/mutate/mutate.go

```diff
@@ -160,7 +163,34 @@ func Config(base v1.Image, cfg v1.Config) (v1.Image, error) {
 	return image, nil
 }
 
-// Created mutates the provided v1.Image to have the provided v1.Time
+// ConfigFile mutates the provided v1.Image to have the provided v1.ConfigFile
+func ConfigFile(base v1.Image, cfg *v1.ConfigFile) (v1.Image, error) {
```

Comment:
There is a lot of code duplication here, it seems. Is it possible to refactor `Config` and `CreatedAt` to use this? (genuine question, I have not given it much thought)

Reply:
I suspect we can yes.

Reply:
Looks great, thanks!